membrane_rtc_engine_hls/test/support/track_sender.ex

```diff
@@ -22,22 +22,26 @@ defmodule Membrane.RTC.Engine.Support.StaticTrackSender do
   alias Membrane.RTC.Engine.Event.TrackVariantResumed
   alias Membrane.RTC.Engine.Track
 
-  def_options track: [
-                type: :struct,
-                spec: Membrane.RTC.Engine.Track.t(),
-                description: "Track this sender will maintain"
-              ]
+  def_options(
```

Comment:
that shouldn't change

Reply:
Yeah, I have some problems with formatting when I `membrane_rtc_engine` in the root directory and from there modify files.